### DISCUSSION TITLE
Recover configuration from previous installation

### DISCUSSION THREAD

**Discussion by Hyphaed** [Frogging-Family/linux-tkg#910] [Q&A] [answered]
When a new kernel base appears fro the froggy family, if I re run the build scripts

is there any possibility to recover the previous selected options?

**Comment by AdelKS** [comment, marked as answer]
You can copy the file https://github.com/Frogging-Family/linux-tkg/blob/master/customization.cfg as `~/.config/frogminer/linux-tkg.cfg` and fill out the variables you want to have always the same

**Reply by Hyphaed** [reply]
many thanks Adel, I will consider to copy the customization.cfg then
